cloud-microphysics/test/main.f90

```diff
@@ -0,0 +1,25 @@
+! Copyright (c), The Regents of the University of California
+! Terms of use are as specified in LICENSE.txt
+program main
+  implicit none
+
+  real t_start, t_finish
+
+  integer :: passes=0, tests=0
+
+  call cpu_time(t_start)
+  block 
+    use netCDF_file_test_m, only : netCDF_file_test_t
```

Comment:
@ktras to play it safe, let's keep all `use` statements at the top of the outermost program unit.
```suggestion
```